cmd/spiffe-helper/main.go

```diff
@@ -35,10 +36,36 @@ func startSidecar(configPath string, exitWhenReady bool, log logrus.FieldLogger)
 	ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt)
 	defer stop()
 
-	spiffeSidecar, err := sidecar.New(configPath, exitWhenReady, log)
+	config, err := hclconfig.ParseConfig(configPath)
 	if err != nil {
-		return fmt.Errorf("Failed to create sidecar: %w", err)
+		return fmt.Errorf("failed to parse %q: %w", configPath, err)
+	}
+	if err := hclconfig.ValidateConfig(config, exitWhenReady, log); err != nil {
+		return fmt.Errorf("invalid configuration: %w", err)
+	}
+
+	sidecarConfig := &sidecar.Config{
+		AddIntermediatesToBundle: config.AddIntermediatesToBundle,
+		AgentAddress:             config.AgentAddress,
+		Cmd:                      config.Cmd,
+		CmdArgs:                  config.CmdArgs,
+		CertDir:                  config.CertDir,
+		ExitWhenReady:            config.ExitWhenReady,
+		JWTBundleFilename:        config.JWTBundleFilename,
+		Log:                      log,
+		RenewSignal:              config.RenewSignal,
+		SvidFileName:             config.SvidFileName,
+		SvidKeyFileName:          config.SvidKeyFileName,
+		SvidBundleFileName:       config.SvidBundleFileName,
+	}
+
+	for _, jwtSvid := range config.JwtSvids {
+		sidecarConfig.JwtSvids = append(sidecarConfig.JwtSvids, sidecar.JwtConfig{
+			JWTAudience:     jwtSvid.JWTAudience,
+			JWTSvidFilename: jwtSvid.JWTSvidFilename,
+		})
 	}
```

Comment:
may we send hclconfig to sidecar.New? so we can unit test that we are parsing it the right way? 
if not, may we move all this code inside hclconfig? (to test it too)

Reply:
Moved the code inside config/ directory. Will add some unit tests for it.

Reply:
Unit tests added.